personal-portfolio/css/style.css

```diff
@@ -750,14 +750,17 @@ button.btn-1:hover {
 
 #icon-1 {
```

Comment:
[ OPTIONAL ] Please add an icon to the `See source` button on the mobile view
![optional](https://user-images.githubusercontent.com/35504149/136590687-6a65a0f5-432d-4bd7-9772-595d7f9cacf5.png)